frontend/Dockerfile

```diff
@@ -2,6 +2,7 @@
 # TODO: can't use vanilla alpine version since python is needed for gql-codegen stuff.
 FROM node:20 AS builder-base
 WORKDIR /app
+COPY . /app/
```

Comment:
what was the purpose of this change? this means none of the layers below will be cached and so even if a ts file changes then the pnpm install will rerun

Reply:
Ah. There's my weak docker understanding blazing forth 😭 
The viewer needs a couple files from `app`, because we're using workspaces.
But, I should just copy the ones it needs - just `pnpm-lock.yaml` and `pnpm-workspace.yaml` I think.

Our `pnpm-lock.yaml` situation is [still a bit imperfect](https://github.com/sillsdev/languageforge-lexbox/pull/882#issue-2350403533), but ultimately, the viewer should need/use the one in the workspace root.

Reply:
I think the viewer was working by building without knowing it was part of a workspace